README.md

```diff
@@ -44,53 +47,70 @@ import cirq
 from mitiq import zne, benchmarks
 
 
-def execute(circuit: cirq.Circuit, noise_level: float = 0.001) -> float:
-    """Returns Tr[ρ |0⟩⟨0|] where ρ is the state prepared by the circuit with depolarizing noise."""
+def execute(circuit, noise_level=0.005):
+    """Returns Tr[ρ |0⟩⟨0|] where ρ is the state prepared by the circuit
+    with depolarizing noise."""
     noisy_circuit = circuit.with_noise(cirq.depolarize(p=noise_level))
-    return cirq.DensityMatrixSimulator().simulate(noisy_circuit).final_density_matrix[0, 0].real
+    return (
+        cirq.DensityMatrixSimulator()
+        .simulate(noisy_circuit)
+        .final_density_matrix[0, 0]
+        .real
+    )
 
 
-circuit: cirq.Circuit = benchmarks.generate_rb_circuits(n_qubits=1, num_cliffords=50)[0]
+circuit = benchmarks.generate_rb_circuits(n_qubits=1, num_cliffords=50)[0]
 
-true_value = execute(circuit, noise_level=0.0)       # Ideal quantum computer.
-noisy_value = execute(circuit)                       # Noisy quantum computer.
-zne_value = zne.execute_with_zne(circuit, execute)   # Noisy quantum computer + Mitiq.
+true_value = execute(circuit, noise_level=0.0)      # Ideal quantum computer
+noisy_value = execute(circuit)                      # Noisy quantum computer
+zne_value = zne.execute_with_zne(circuit, execute)  # Noisy quantum computer + Mitiq
 
-print(f"Error (w/o  Mitiq): %0.4f" %abs((true_value - noisy_value) / true_value))
-print(f"Error (with Mitiq): %0.4f" %abs((true_value - zne_value) / true_value))
+print(f"Error w/o  Mitiq: {abs((true_value - noisy_value) / true_value):.3f}")
+print(f"Error w Mitiq:    {abs((true_value - zne_value) / true_value):.3f}")
 ```
+
 Sample output:
+
 ```
-Error (w/o  Mitiq): 0.0688
-Error (with Mitiq): 0.0002
+Error w/o  Mitiq: 0.264
+Error w Mitiq:    0.073
 ```
 
+### Calibration
+
+Unsure what error mitigation technique or parameters to use?
+Try out the calibration module demonstrated below to help find the best parameters for you particular backend!
+
+<video width="520" height="400" controls>
```

Comment:
Great to embed the video! Unfortunately I don't see it when I view this version of the readme.

Reply:
Good catch, I replaced it with an uploaded movie in [513392c](https://github.com/unitaryfund/mitiq/pull/1778/commits/513392c697fabf68b4bf77eb8eecf5a133dfa561) which works on github, but not RTD. Using a gif ([48d84d2](https://github.com/unitaryfund/mitiq/pull/1778/commits/48d84d2de9c74e49b0489ff24a12e7016011618a)), seems to work for both.